src/main/java/edu/illinois/cs/dt/tools/minimizer/MinimizeTestsResult.java

```diff
@@ -73,33 +69,37 @@ public boolean verify(final Runner runner) throws Exception {
     }
 
     public boolean verify(final Runner runner, final int verifyCount) throws Exception {
-        for (int i = 0; i < verifyCount; i++) {
-            final List<List<String>> depLists = ListUtil.sample(ListUtil.subsequences(deps), MAX_SUBSEQUENCES);
-            int check = 1;
-            int totalChecks = 2 + depLists.size() - 1;
-
-            IOUtil.printClearLine(String.format("Verifying %d of %d. Running check %d of %d.", i + 1, verifyCount, check++, totalChecks));
-            // Check that it's correct with the dependencies
-            if (!isExpected(runner, deps)) {
-                throw new MinimizeTestListException("Got unexpected result when running with all dependencies!");
+        for (PolluterData polluter : polluters) {
```

Comment:
I think this process should also be updated, because right now if any polluter fails verification, the entire process will crash, even though all the other polluters we found may be fine. We should probably separate it out/catch the MinimizeTestListException to filter out only the polluters that cannot be verified.

Reply:
Yes, I've modified the code here to use some try+catch within the loop. Maybe we need to add a bit more logic to actually throw an exception out in case something really goes bad?

Reply:
It seems that we don't actually remove/mark polluters as failing verification when they fail, only print out a message?

RE throwing an exception, I'm not sure what case would be best served by an exception? It seems to me that the worst case is that we find that none of the polluters can be verified, in which case we simply report no polluters. In that case, none of the following steps should execute anyway, correct?

Reply:
Good point, I've added the code that removes polluters that throw the exception.